components/product-list/ProductList.js

```diff
@@ -10,37 +11,48 @@ export default class ProductList extends Component {
     title: 'Produtos'
   }
 
+  constructor(props) {
+    super(props)
+
+    const { navigation } = this.props
+
+    this.state = {
+      user: navigation.getParam('user'),
+      products: navigation.getParam('products')
```

Comment:
Acho que a gnete tinha concordado em colocar `,` no ultimo elemento de um array/objeto antes de pular linha pra fechar, pra manter o dono da linha quando alguem adicionar um novo elemento no final...

```suggestion
      products: navigation.getParam('products'),
```

Reply:
A gnete tinha concordado sim, no momento o @castilh0s tá trabalhando na issue do linter que vai resolver todos os nossos problemas!

Reply:
Fato. Acredito que podemos esperar o PR #28 ser mesclado e aí posso corrigir as alterações com base nos padrões configurados do eslint.